das/google_cloud_storage_service.go

```diff
@@ -61,90 +67,56 @@ func (g *GoogleCloudStorageClient) Close(ctx context.Context) error {
 }
 
 type GoogleCloudStorageServiceConfig struct {
-	Enable       bool          `koanf:"enable"`
-	AccessToken  string        `koanf:"access-token"`
-	Bucket       string        `koanf:"bucket"`
-	ObjectPrefix string        `koanf:"object-prefix"`
-	EnableExpiry bool          `koanf:"enable-expiry"`
-	MaxRetention time.Duration `koanf:"max-retention"`
+	Enable              bool   `koanf:"enable"`
+	AccessTokenFile     string `koanf:"access-token-file"`
```

Comment:
Good to leave support for `access-token` as well as add support for `access-token-file` for maximum flexibility